ios/Video/RCTVideo.swift

```diff
@@ -1329,11 +1326,11 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     }
 
     func setLicenseResult(_ license: String!, _ licenseUrl: String!) {
```

Comment:
@KrzysztofMoch I am reviewing use of String! and it is a bad practice and can lead to some issues, see: https://github.com/TheWidlarzGroup/react-native-video/issues/4181

It is just hide the warning, so no need to test the value ...
I think it would be better to change like I propose (not sure of the consequences inside _drmManager)

```suggestion
    func setLicenseResult(_ license: String?, _ licenseUrl: String?) {
```

Reply:
I just changed it to `String` - I don't like idea of making it `String?` as it gives idea that those params are not required when they are

Reply:
But the app can call it with undefined parameter, in that case we should be robust. Am I wrong ?
This is not a mandatory change!

Reply:
Alright then we should made changes here (but lets do it in other PR):
https://github.com/TheWidlarzGroup/react-native-video/blob/c96f7d41f38499baefde2b7fa7cf10441463438e/ios/Video/RCTVideoManager.swift#L40-L45